pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/db/dto/FormBlockDto.java

```diff
@@ -2,26 +2,34 @@
 
 import lombok.AllArgsConstructor;
 import lombok.Value;
-import java.sql.ResultSet;
-import java.sql.SQLException;
-
-import org.broadinstitute.ddp.constants.SqlConstants;
 
 import org.broadinstitute.ddp.model.activity.types.BlockType;
-import org.jdbi.v3.core.mapper.RowMapper;
-import org.jdbi.v3.core.statement.StatementContext;
+import org.broadinstitute.ddp.model.pex.Expression;
+import org.jdbi.v3.core.mapper.Nested;
+import org.jdbi.v3.core.mapper.reflect.ColumnName;
+import org.jdbi.v3.core.mapper.reflect.JdbiConstructor;
 
 /**
  * A wrapper data object around a block and its optional conditional expression.
  */
 @Value
-@AllArgsConstructor
+@AllArgsConstructor(onConstructor = @__({@JdbiConstructor}))
```

Comment:
`{` and `}` might be avoided here since `JdbiConstructor` is  the only one propagated annotation